### ISSUE TITLE
I couldn't configure  Pyright.disableProgressNotifications

### ISSUE THREAD

**Issue by QianquanChina** [fannheyward/coc-pyright#764]
## Problem
I used the `:CocUpdateSync` command to update the Coc-Pyright but I couldn't configure this  
`Pyright.disableProgressNotifications` in coc-setting.json. 

![](https://s3.bmp.ovh/imgs/2022/06/08/82a325dc2beee154.png)
why? Thanks.
### `:CocCommand pyright.version`

```
[coc.nvim] coc-pyright 1.1.252 with Pyright 1.1.252
```
### `:CocCommand workspace.showOutput Pyright`
```
Workspace: /home/jc  
Using python from /home/jc/.local/share/anaconda3/envs/PyTorch/bin/python
[Warn  - 上午10:38:50] Ignored rootPath /home/jc of client "pyright"                                                                                 
[Warn  - 上午10:38:50] No valid workspaceFolder exists                                                                                                
[Info  - 上午10:38:51] Pyright language server 1.1.252 starting                                                                                        
[Info  - 上午10:38:51] Server root directory: /home/jc/.config/coc/extensions/node_modules/coc-pyright/node_modules/pyright/dist/                      
[Info  - 上午10:38:51] No pyproject.toml file found.                                                                                                   
[Info  - 上午10:38:51] Setting pythonPath for service "<default>": "/home/jc/.local/share/anaconda3/envs/PyTorch/bin/python"                           
[Warn  - 上午10:38:51] stubPath typings is not a valid directory.                                                                                      
[Info  - 上午10:38:51] Assuming Python version 3.8                                                                                                     
[Info  - 上午10:38:51] Assuming Python platform Linux                                                                                                 
[Info  - 上午10:38:51] Searching for source files                                                                                                     
[Info  - 上午10:38:51] No source files found.
```

**Comment by yaegassy** [comment]
@QianquanChina The package containing the `pyright.disableProgressNotifications` feature has not yet been published to npm.

Perhaps as soon as there is a next release of the pyright's language server, the `coc-pyright` package will be published to npm.

**Comment by QianquanChina** [comment]
@yaegassy  Thanks.

**Comment by fannheyward** [comment]
Released in v1.1.253.
